### PR TITLE
Adding the complaints policy to the code of conduct

### PR DESCRIPTION
**Pull request by NataliaLKB**

### CODE REVIEW

File: code_of_conduct.md [hunk 1]
```diff
@@ -27,5 +27,6 @@ Particularly in the case of physical events, organisers will be happy to help pa
 
 We expect participants to follow these rules in all community interactions, whether professional, social or a mix of both.
 
-[Complaints form](https://docs.google.com/forms/d/e/1FAIpQLSd8Xro2JoVJ2_GUtILtX-Hfwlwkb3P3PSKabTPHV29BKqrPYg/viewform).
+If someone has violated these rules, please review the [Complaints Policy](https://github.com/foundersandcoders/london-membership/blob/master/complaints-policy.md) and make a complaint using out
```
Comment (claireinez) [inline code_of_conduct.md]: Typo: "make a complaint using _our_"

Comment (NataliaLKB) [inline code_of_conduct.md]: Thanks @claireinez, all fixed now!

**Comment by bradreeder** [comment]
Two reviewers approved. All good! 👍

**Comment by NataliaLKB** [comment]
Thanks @bradreeder and @iteles !

**Pull request merged**